extensions/aerospace/src/utils/appSwitcher.tsx

```diff
@@ -28,13 +27,25 @@ function getAppPath(bundleId: string) {
 
   return appPath.stdout.trim();
 }
+// aerospace list-windows --json --workspace focused --format "%{app-name} %{window-title} %{window-id} %{app-pid} %{workspace} %{app-bundle-id}"
 
 export function getWindows() {
-  const aerospaceArr = spawnSync("sh", [`${environment.assetsPath}/scripts/list-windows.sh`], {
```

Comment:
So we don't need the shell script anymore, right?

Reply:
As @nikitabobko suggested, I inlined the script instead of using an external .sh file.